workshops/test/test_util.py

```diff
@@ -376,195 +374,184 @@ def test_attendance_increases(self):
+            'start': None,
+            'end': None,
+            'country': '',
             'venue': '',
             'address': '',
-            'country': '',
-            'latitude': '',
-            'longitude': '',
+            'latitude': None,
+            'longitude': None,
+            'reg_key': None,
+            'instructors': [],
+            'helpers': [],
+            'contact': '',
         }
+        self.assertEqual(expected, parse_tags_from_event_website(empty_dict))
 
-        self.assertEqual(parse_tags_from_event_index(url), expected)
-
-    @patch.object(requests, 'get')
-    def test_parsing_empty_list_values(self, mock_get):
-        mock_get.return_value.status_code = 200
-        mock_get.return_value.text = """---
-venue: Euphoric State University
-address: Highway to Heaven 42, Academipolis
-country: USA
-language: us
-latlng: 36.998977, -109.045173
-humandate: Jul 13-14, 2015
-humantime: 9:00 - 17:00
-startdate: 2015-07-13
-enddate: 2015-07-14
-instructor:  # instructors
-helper:  # helpers
-contact: hermione@granger.co.uk, rweasley@ministry.gov.uk
-etherpad:
-eventbrite: 10000000
----
-"""
-        url = 'http://test.github.io/2015-07-13-test/'
-        notes = """INSTRUCTORS: \n\nHELPERS: \n\nCOUNTRY: USA"""
-
+    def test_parsing_correct_tags(self):
+        tags = {
+            'slug': '2015-07-13-test',
+            'startdate': '2015-07-13',
+            'enddate': '2015-07-14',
+            'country': 'us',
+            'venue': 'Euphoric State University',
+            'address': 'Highway to Heaven 42, Academipolis',
+            'latlng': '36.998977, -109.045173',
+            'language': 'us',
+            'instructor': 'Hermione Granger, Ron Weasley',
+            'helper': 'Peter Parker, Tony Stark, Natasha Romanova',
+            'contact': 'hermione@granger.co.uk, rweasley@ministry.gov',
+            'eventbrite': '10000000',
+        }
         expected = {
             'slug': '2015-07-13-test',
-            'start': date(2015, 7, 13),
-            'end': date(2015, 7, 14),
-            'url': 'https://test.github.io/2015-07-13-test/',
-            'reg_key': 1e7,
-            'contact': 'hermione@granger.co.uk, rweasley@ministry.gov.uk',
-            'notes': notes,
+            'language': 'US',
+            'start': datetime.date(2015, 7, 13),
+            'end': datetime.date(2015, 7, 14),
+            'country': 'US',
             'venue': 'Euphoric State University',
             'address': 'Highway to Heaven 42, Academipolis',
+            'latitude': 36.998977,
+            'longitude': -109.045173,
+            'reg_key': 10000000,
+            'instructors': ['Hermione Granger', 'Ron Weasley'],
+            'helpers': ['Peter Parker', 'Tony Stark', 'Natasha Romanova'],
+            'contact': 'hermione@granger.co.uk, rweasley@ministry.gov',
+        }
+        self.assertEqual(expected, parse_tags_from_event_website(tags))
+
+    def test_parsing_tricky_tags(self):
```

Comment:
Probably better to have separate tests for each tricky case, and then have more descriptive names for each case.  If we have a single tricky-tags test we only get a single tricky case for each tag.